### PR TITLE
Added script to validate and publish packages to npm for the new release process

### PR DESCRIPTION
**Pull request by psmyrek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature (release-tools): Added script to validate and publish packages to npm. Closes ckeditor/ckeditor5#13957.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: packages/ckeditor5-dev-release-tools/lib/tasks/publishpackages.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * @param {String} [options.cwd=process.cwd()] Current working directory from which all paths will be resolved.
+ * @returns {Promise}
+ */
+module.exports = async function publishPackages( options ) {
+	const {
+		packagesDirectory,
+		npmOwner,
+		npmTag = 'staging',
+		optionalEntries = null,
+		confirmationCallback = null,
```
Comment (martnpaneq) [inline packages/ckeditor5-dev-release-tools/lib/tasks/publishpackages.js:47]: I have a question about `confirmationCallback`. If I understand correctly, it is supposed to work as a dry run? Or do we have other use case for it?

**Review by martnpaneq — APPROVED** [review]
I really like that utils are separated from the `publishPackages` function. It's very easy to read :)

File: packages/ckeditor5-dev-release-tools/lib/tasks/publishpackages.js (hunk 1 above)
Comment (psmyrek) [reply packages/ckeditor5-dev-release-tools/lib/tasks/publishpackages.js:47]: The purpose of this callback is to give user the final opportunity to confirm or cancel the publishing on npm.

For manually invoked scripts, I would expect a prompt here asking the user whether to continue (and that there will be no turning back after confirmation). For a script invoked on CI that should publish nightly versions, the callback will probably always return `true`.

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**